web/client/actions/map.js

```diff
@@ -31,6 +31,7 @@ export const MOUSE_OUT = 'MOUSE_OUT';
 export const MAP_PLUGIN_LOAD = 'MAP:MAP_PLUGIN_LOAD';
 export const ORIENTATION = 'MAP:ORIENTATION';
 export const UPDATE_MAP_VIEW = 'MAP:UPDATE_MAP_VIEW';
+export const UPDATE_SCENE_CONFIG = 'MAP:UPDATE_SCENE_CONFIG';
```

Comment:
Could we call this `UPDATE_MAP_OPTIONS` instead? Looking to the reducer it seems not strictly related to the scene so I would suggest to keep this action related to all the map options.